### DISCUSSION TITLE
Is there a way to change a scaling threshold during runtime from scaler code?

### DISCUSSION THREAD

**Discussion by spoplavskiy** [kedacore/keda#1229] [General]
I'm curios, can I somehow trigger a reconciling process for scaler from code, if I want to change a scaler threshold? 
The scenarios could be:

1. For Log Analytics and Prometheus scalers retrieve and set threshold dynamically based on query result.
2. Change thresholds dynamically based on day\night or other schedules.

**Comment by zroubalik** [comment]
I am not sure that you can do that, but what do you expect to happen? The reconciliation process always looks at the ScaledObject definition deployed in the cluster and get the values from there. So in case you are able somehow kick the reconciliation process from code, the treshold will be always taken from the ScaledObject spec.


1. For this I feel that the best place to implement these features are the scalers itself.
2. dtto, or maybe try to use a cron scaler?
